package.json

```diff
@@ -6,22 +6,21 @@
   "license": "BSD-2-Clause",
   "version": "3.14.1",
   "engines": {
-    "node": ">=4.0.0"
+    "node": ">=6.0.0"
   },
   "maintainers": [
     "Fábio Santos <fabiosantosart@gmail.com>",
     "Alex Lam <alexlamsl@gmail.com>",
     "Mihai Bazon <mihai.bazon@gmail.com> (http://lisperator.net/)"
   ],
   "repository": "https://github.com/fabiosantoscode/terser.git",
-  "main": "dist/bundle.js",
+  "main": "dist/bundle.min.js",
```

Comment:
Not sure, is it good idea to use minified bundles for node.js modules ? It will be hard to debug build when it fail on terser phase, am I wrong?